db/migrate/20210726183045_change_to_utf8_mb4.rb

```diff
@@ -44,7 +44,7 @@ def self.up
     execute "ALTER TABLE groups CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_bin"
     execute "ALTER TABLE groups_users CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_bin"
     execute "ALTER TABLE identities CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_bin"
-    execute "ALTER TABLE keywords CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_bin"
+#    execute "ALTER TABLE keywords CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_bin"
```

Comment:
Better to just nuke these two lines rather than level the references around.  Keywords are gone gone, and not coming back.